src/core/dbus-cgroup.c

```diff
@@ -320,6 +320,31 @@ static int property_get_ip_address_access(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_ip_filters(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+        char*** filters = userdata;
+        char **entry;
+        int r;
+
+        r = sd_bus_message_open_container(reply, 'a', "s");
```

Comment:
use sd_bus_message_append_strv() instead, it does the same thing in one go. That said, you can actually drop the whole function, since sd-bus does such a serialization anyway implicitly if it encounters an "as" property in the vtable, and the getter is set to NULL. Hence, please use that, drop this getter routine and just pass NULL. It works the same with zero code.